imports/plugins/core/accounts/server/no-meteor/resolvers/Query/groups.js

```diff
@@ -11,19 +11,24 @@ import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/sh
  */
 
 /**
- * @name "Query.groups"
+ * @name Query/groups
  * @method
  * @memberof Accounts/GraphQL
  * @summary find and return the administrators (users with "admin" or "owner" role) for a shop
  * @param {Object} _ - unused
  * @param {GroupConnectionArgs} args - an object of all arguments that were sent by the client. {@link ConnectionArgs|See default connection arguments}
  * @param {Object} context - an object containing the per-request state
+ * @param {Object} info Info about the GraphQL request
  * @return {Promise<Object[]>} an array of user Group objects
  */
-export default async function groups(_, { shopId, ...connectionArgs }, context) {
+export default async function groups(_, { shopId, ...connectionArgs }, context, info) {
   // Transform ID from base64
   const dbShopId = decodeShopOpaqueId(shopId);
 
   const query = await context.queries.groups(context, dbShopId);
-  return getPaginatedResponse(query, connectionArgs);
+  return getPaginatedResponse(query, connectionArgs, {
+    includeHasNextPage: wasFieldRequested("pageInfo.hasNextPage", info),
```

Comment:
Would it be easier to maintain if we do this inside of `getPaginatedResponse.js` instead of passing it individually in every `getPaginatedResponse`?

Reply:
I did consider that, but I wanted to try to keep that function unaware of GraphQL. That way it could be used for some other interface, too. (In fact, GraphQL info isn't available in a couple places that call it.) But I agree it feels kind of yucky to repeat similar code everywhere. One idea might be to make a GraphQL-aware lightweight wrapper function. Maybe I'll do that.

Reply:
Makes sense either way. I figured there was some reasoning behind it, just noticing the same code in 27 places I wanted to make sure the question was asked.